src/components/molecules/CollateralProgressBar/CollateralProgressBar.test.tsx

```diff
@@ -25,7 +26,7 @@ describe('CollateralProgressBar Component', () => {
         );
     });
 
-    it('should render zero collateral CollateralProgressBar', () => {
+    it('should render zero collateral CollateralProgressBar', async () => {
```

Comment:
In these tests, the content inside the tooltip is more important than the tooltip itself.
We must bring back these assertions.